client/src/app/pages/identities/components/identity-form/identity-form.tsx

```diff
@@ -637,6 +639,35 @@ export const IdentityForm: React.FC<IdentityFormProps> = ({
         </>
       )}
 
+      {values?.kind === "jira" && (
```

Comment:
Since we are using the same form validation between jira & proxy... Wondering if we can use the same form fields also?